docs/HelpStyleGuide.md

```diff
@@ -0,0 +1,68 @@
+
+* `*Italics*`
+* `**Bold**`
+* `***Italics and Bold***`
+* `# Heading 1` up to `###### Heading 6`
+  * Use `#H1` for the name of the module at the beginning of the help text
+  * Use `#H2` for any subheadings
+  * Use `#H4` for sub-subheadings
+* Avoid using HTML line break (`<br>`) tags—simply use physical line breaks (spaces between lines) where necessary. 
+  * Using `<br>` might be necessary only in lists of items
+  
+  ## When to Use Emphasis
+  
+Whether a user knows it or not, they will come to recognize styles that are consistent throughout the text. This way, their eyes can jump to a part of the text that addresses their issue, if they don’t need to read the whole thing. Follow these rules on using emphasis:
+
+* *Italicize* when referring to a specific place on the screen
+  * i.e. in the Selection Filters section
+  * i.e. in the Metadata column of the table
+  * i.e. in the Imaging menu
+* **Bold** when referring to any clickable button, including modules and tabs
+  * DON’T call it a button
+    * i.e. Click **Submit** (rather than “click the **Submit** button”)
+  * DO call it a module or tab
+    * i.e. In the **Imaging Browser** module
+    * i.e. Click the **Upload** tab
+  * You can also use bold when defining features in a list, to separate the term being defined from the definition itself, i.e. when describing columns in a table:<br>
+     **Column heading 1**: description<br>
+     **Column heading 2**: description<br>
+     Etc.
+     
+## Tone
+
+LORIS is used by scientific community worldwide, including countries where English is a second, or even third language. Therefore, it is important that LORIS documentation (help text, setup guides, etc.) be written in a way that it comprehensible by people with limited knowledge and understanding of English language and expression.
+
+Please note, this is NOT about “dumbing things down”. It’s simply about using the most direct language and tone to get the information across to the user in the least amount of time, with the least amount of friction. The quicker a user understands something, the happier they are, the more trust they have with the software, and the less silly questions you can expect to get to the dev mailing list! 
+
+* Use second person point of view rather than third
+  * i.e. “This module allows *you* to browse…” (rather than “This module allows *users* to browse…”)
+* Keep it concise. Avoid unnecessary information.
+  * Try not to be repetitive. 
+  * i.e. the Help text for the Imaging Browser module begins with the title Imaging Browser. You don’t need to start text with “In the Imaging Browser module...”, because the user already knows the module based on the title. Instead, keep it short and say “In this module…”
+* Assume the user is tech-savvy enough in the year 2020 to know about features that are intuitive or common across modern platforms, so you don’t need to repeat in every Help text
```

Comment:
```suggestion
* Assume the user is tech-savvy enough to know about features that are intuitive or common across modern platforms, so you don’t need to repeat in every Help text
```

I see why this was added stylistically but I know that we'll forget to keep this current.